hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/table/action/compact/HoodieFlinkMergeOnReadTableCompactor.java

```diff
@@ -73,10 +73,11 @@ public List<WriteStatus> compact(HoodieCompactionHandler compactionHandler,
                                    CompactionOperation operation,
                                    String instantTime,
                                    TaskContextSupplier taskContextSupplier,
-                                   Option<HoodieReaderContext<?>> readerContextOpt) throws IOException {
+                                   Option<HoodieReaderContext<?>> readerContextOpt,
+                                   HoodieTable table) throws IOException {
```

Comment:
If parameter`table` is necessary, `metaClient` in above can be removed and using `table.getMetaClient`.